theme/themes/pastanaga/extras/blocks.less

```diff
@@ -470,18 +469,19 @@ body.has-toolbar.has-sidebar-collapsed .ui.wrapper > .ui.inner.block.full {
 }
 
 .ui.basic.button.block-add-button {
-  // .ui.basic.button.block-delete-button {
   position: absolute;
   z-index: 2;
-  // margin-top: -21px;
-  top: -2px;
-  padding: 0;
-  border: none;
+  bottom: -22px;
+  left: 50%;
+  padding: 0 !important;
```

Comment:
IMHO, Changing the padding here could improve the UI a little:

```
0 1rem !important
```

![plus-button](https://user-images.githubusercontent.com/323385/200546705-4e5e3ebc-a091-4758-bcd1-bb1127748311.png)

Reply:
@avoinea I think that looks fine, but it looks odd if the page has a different background color, because of the way I'm using border-radius to create a circle around the icon. I'm not sure how to create this space while matching the background color, so I think I'll leave it as is for now.